.github/CODEOWNERS

```diff
@@ -7,22 +7,27 @@
 # be requested for review when someone opens a pull request.
 # order is alphabetical for easier maintenance.
 #
-# Sean Gallacher (gallacher)
-# Trevor Dawe (tdawe)
+# AnandatDell (anandrajak1)
 # Alexander Hoppe (hoppea2)
 # Alik Saring (alikdell)
 # Aaron Tye (atye)
+# Chiman Jain (chimanjain)
+# Don Khan (donatwork)
 # Florian Coulombel (coulof)
+# Harish H (HarishH-DELL)
+# Nitesh Rewatkar (nitesh3108)
+# Rajendra Indukuri (rajendraindukuri)
+# Sean Gallacher (gallacher)
 # Shayna Finocchiaro (shaynafinocchiaro)
 # Sharmila Ramamoorthy (sharmilarama)
+# Shefali Malhotra (shefali-malhotra)
+# Spandita Panigrahi (panigs7)
 # Forrest Xia (forrestxia)
+# Trevor Dawe (tdawe)
 # Yian Zong (YianZong)
 # Yiming Bao (baoy1)
 # Tao He (taohe1012)
 # Peter Cao (P-Cao)
-# Don Khan (donatwork)
-# Harsha Yalamanchili (harshaatdell)
-# AnandatDell (anandrajak1)
 
 # for all files:
-* @gallacher @tdawe @alikdell @atye @hoppea2 @coulof @shaynafinocchiaro @sharmilarama @forrestxia @YianZong @baoy1 @taohe1012 @P-Cao @donatwork @harshaatdell @anandrajak1
+* @gallacher @tdawe @alikdell @atye @hoppea2 @coulof @shaynafinocchiaro @sharmilarama @forrestxia @YianZong @baoy1 @taohe1012 @P-Cao @donatwork @anandrajak1 @chimanjain @HarishH-DELL @nitesh3108 @rajendraindukuri @shefali-malhotra @panigs7 
```

Comment:
If you are touching the CODEOWNERS then you should reduce the members to a few people, two from each owning team owning this component. Remove the others. Do this for all repos based on ownership. Don't be shy to remove members.

Reply:
Sure @donatwork, I'll update it 2 per team to keep it clean. Thanks.

Reply:
updated from celestials team, can you please recommend from other's team.